requirements_dev.txt

```diff
@@ -0,0 +1,3 @@
+bumpversion==0.5.3
+wheel>=0.31.1
```

Comment:
for consistency with other openwisp modules, could you please rename to `requirements-dev.txt`?

Reply:
ok